sql-plugin/src/main/311until330-all/scala/com/nvidia/spark/rapids/shims/GpuTypeShims.scala

```diff
@@ -120,4 +124,5 @@ object GpuTypeShims {
    */
   def additionalCommonOperatorSupportedTypes: TypeSig = TypeSig.none
 
+  def isDayTimeType(t: DataType): Boolean = false
```

Comment:
The name here is problematic on Spark 3.2.x which does support `DayTimeIntervalType` and could return false when called with a `DayTimeIntervalType` instance.  Maybe this should be named something like `isSupportedDayTimeType`?

Reply:
done,  already have `isSupportedDayTimeType`.